src/style.css

```
@@ -0,0 +1,20 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+.article-title {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+}
+
+.article {
+  font-size: 15px;
+  font-weight: 100;
```

better to use this one
```suggestion
  font-weight: 400;
```